### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by justinvdm**
Cleaning up the docs after the first pass at designing and documenting the API.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: docs/index.rst [hunk 1]
```diff
@@ -1541,7 +1343,7 @@ It is also possible to create revisions in bulk by providing an array of revisio
        }
      }]
 
-.. note
+.. note::
   Created revisions in bulk is done atomically. If one of the given revisions cannot be created, none of the given revisions will be created.
```
Comment (hodgestar) [inline docs/index.rst]: s/Created/Creating/ ?

**Comment by hodgestar** [comment]
One minor unrelated typo, otherwise looks good.

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**